### PR TITLE
add first round of personas to glossary

### PR DESCRIPTION
**Pull request by fkr**
Discussed in PB on Nov 28th

Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

### CODE REVIEW

File: GLOSSARY.md [hunk 1]
```diff
@@ -124,3 +124,13 @@ Swift is the so-called object storage that can be used by Nova. This is responsi
...
 A Ceph OSD (Object Storage Daemon) logically represents a storage device in a Ceph cluster, which can logically be a hard disk, which is the ideal case. In other cases it can also be a raid, which however leads to considerable performance limitations due to caching or other raid optimization.
+
+## Personas
+
+|Persona|Description|
+|SCS Operator|The SCS Operator is the one who owns and operates a standardized cloud environment.|
+|SCS Consumer|The SCS Consumer consumes a standardized SCS environment and operates and orchestrates applications on top of it. The SCS Consumer is typically a customer or user of the SCS Operator.|
+|SCS Integrator|The SCS Integrator assists in or is building up a standardized cloud environment. The SCS Integrator can be 2nd or 3rd level support for the SCS Operator.|
+|SCS Project|The SCS Project oversees the overall activities around the Sovereign Cloud Stack.|
+|SCS Developer|The SCS Developer actively contributes to technical elements of the Sovereign Cloud Stack.|
```
Comment (maxwolfs) [inline GLOSSARY.md:135]: What kind of piped formatting is this? I have corrected some sentences and added a list for better readability.

```suggestion
|Persona|Description|

- |SCS Operator|The SCS Operator owns and operates a standardized cloud environment.|
- |SCS Consumer|The SCS Consumer consumes a standardized SCS environment, operates and orchestrates applications on top of it. The SCS Consumer is typically a customer or user of the SCS Operator.|
- |SCS Integrator|The SCS Integrator assists in or is building up a standardized cloud environment. The SCS Integrator can be 2nd or 3rd level support for the SCS Operator.|
- |SCS Project|The SCS Project oversees the overall activities around the Sovereign Cloud Stack.|
- |SCS Developer|The SCS Developer actively contributes to technical elements of the Sovereign Cloud Stack.|
```

Comment (fkr) [reply GLOSSARY.md]: sorry, i botchered the table. fix with the next commit.

**Review by maxwolfs — DISMISSED** [review]
lgtm!

**Comment by garloff** [comment]
On SCS Operator:
We probably think about two slightly different things when we hear "SCS Operator":
- The company that runs / operates an SCS compliant cloud/container stack (with all its functions: Development, Operations, Support, Marketing, Sales, G&A)
- The Operations team of the SCS operating company.
(And maybe something in between: The technical teams for DevOps and Support of the SCS operating company -- which would also roughly match the IT department in a company that runs the SCS compliant stack as private cloud.)
We probably should clarify this.

**Comment by fkr** [comment]
> On SCS Operator: We probably think about two slightly different things when we hear "SCS Operator":
> 
> * The company that runs / operates an SCS compliant cloud/container stack (with all its functions: Development, Operations, Support, Marketing, Sales, G&A)
> * The Operations team of the SCS operating company.
> 
> (And maybe something in between: The technical teams for DevOps and Support of the SCS operating company -- which would also roughly match the IT department in a company that runs the SCS compliant stack as private cloud.)
> 
> We probably should clarify this.

I actually fear that by over specifying it, it will loose focus and we will focus on non-important details.
Nevertheless, to address your points @garloff, I'd suggest to then divide the SCS Operator into:

SCS Operator
SCS Operations Team

**Review by garloff — APPROVED** [review]
/lgtm

**Comment by fkr** [comment]
thanks for your feedback @garloff. Always good to have second opinions! :)

**Pull request merged**